src/actions/formActions.js

```diff
@@ -20,3 +23,18 @@ export const fetchFormCategories = () =>
       dispatch(fetchFormCategoriesSuccess(data));
     }
   );
+
+const createFormRequest = createAction(CREATE_FORM);
+const createFormSuccess = createAction(CREATE_FORM_SUCCESS);
+const createFormError = createAction(CREATE_FORM_ERROR);
+
+export const createForm = (values, history) =>
+  swallow(
+    (error, dispatch) => dispatch(createFormError(error)),
+    async dispatch => {
+      dispatch(createFormRequest());
+      const { data } = await formServices.createForm(values);
+      dispatch(createFormSuccess(data));
+      history.push('/forms');
```

Comment:
is there a route constant defined for this? I'd rather not have it hardcoded.

Reply:
I didn't define it yet because the routes doesn't exist, will work on it on the next feature where I have to display a list of forms

Reply:
ok, doing it